### PR TITLE
fix: swap leveldown/level.js for level

### PR DESCRIPTION
**Pull request by achingbrain**
The `level` module bundles `leveldown`, `encoding-down` and `levelup`, as well as substituting `level.js` for `leveldown` in the browser.

This change uses `level` over all those other modules.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,22 +15,21 @@ class LevelDatastore {
...
-        }
-      }
-    )
+    this.db = database(path, {
+      ...opts,
+      valueEncoding: 'binary',
+      compression: false // same default as go
+    })
+
+    this.db.put = promisify(this.db.put)
```
Comment (vweevers) [inline src/index.js]: Why is this needed? In case a custom db (opts.db) is passed in?

Comment (achingbrain) [reply src/index.js]: I was seeing errors like `'get() requires a callback argument'` ...and now I'm not.  Have removed.
